scopes/lanes/ui/gallery/lane-gallery.tsx

```diff
@@ -64,7 +59,12 @@ function LaneGalleryWithPreview({ currentLane, overviewItems, routeSlot, host }:
             componentUrl={LanesModel.getLaneComponentUrl(component.id, currentLane.id)}
           />
         )
-      : ({ component }) => <ScopeComponentCard component={component} />;
+      : ({ component }) => (
+          <ScopeComponentCard
+            component={component}
+            componentUrl={LanesModel.getLaneComponentUrl(component.id, currentLane.id)}
```

Comment:
why not call it `href` and that way it keeps the api consistent with `ComponentCard`?